epictrack-web/src/components/proponent/ProponentList.tsx

```diff
@@ -255,6 +266,19 @@ export default function ProponentList() {
                     Create Proponent
                   </Button>
                 </Restricted>
+                <Tooltip title="Export to csv">
+                  <IconButton
+                    onClick={() =>
+                      exportToCsv({
+                        table,
+                        downloadDate: new Date().toISOString(),
+                        filenamePrefix: "proponent-listing",

```

Comment:
Can you give me a screen shot of how this would look in the table

Reply:
this is a csv visualizer 
![image](https://github.com/bcgov/EPIC.track/assets/103138766/07a46bed-e24b-4b02-9a3c-90e3e1a2b482)